Source/WarpX.cpp

```diff
@@ -779,54 +743,22 @@ WarpX::AllocLevelData (int lev, const BoxArray& ba, const DistributionMapping& d
     }
 
     if (n_current_deposition_buffer < 0) {
-        n_current_deposition_buffer = ngJ.max();
+        n_current_deposition_buffer = nJ_buffer;
```

Comment:
See earlier comment on the return value of the constructor of `guardCellManager`.

Reply:
Sure, done.